src/pi_FPE/requirements.txt

```diff
@@ -1,4 +1,5 @@
 scipy
 numpy
 pandas
-termcolor
\ No newline at end of file
+termcolor
+pyyaml
```

Comment:
No need of pyyaml anymore.

Reply:
I could verify, but I add to insert it in the docker image otherwise some component did not work. 

fooplacementestimator does use yaml

Reply:
If I take out the pyyaml, I get:

```term
docker run --rm -v $PWD/tests/data/input:/in -v $PWD/out_tests:/out pi_udbenchmark run_fpe /in/subject_04_cond_23_run_01_com.csv /in/subject_04_cond_23_run_01_angMomentum.csv /in/subject_04_cond_23_run_01_comItensor.csv /in/subject_04_cond_23_run_01_gaitEvents_01 /in/subject_04_cond_23_run_01_testbedLabel_01.yaml /out

Traceback (most recent call last):
  File "/usr/local/bin/run_fpe", line 3, in <module>
    from pi_FPE.footplacementestimator import main
  File "/usr/local/lib/python3.7/site-packages/pi_FPE/__init__.py", line 2, in <module>
    from .footplacementestimator import calcFPE
  File "/usr/local/lib/python3.7/site-packages/pi_FPE/footplacementestimator.py", line 12, in <module>
    import yaml

```